### DISCUSSION TITLE
VNM- Console stuck in caps lock mode

### DISCUSSION THREAD

**Discussion by lizannea** [neurodesk/neurodesk.github.io#140] [Q&A] [answered]
After running VNM using the instructions on a Mac. I opening the terminal within the virtual desktop and tried to paste a line in the terminal to log in on OzStar. After doing this my console has been stuck in caps lock mode.
 
At the moment I can only type something in lower caps if I keep the shift key down. This is only the case in the terminal within the virtual desktop. 
 
Someone mentioned it could be fixed by keeping the shift key pressed while you write the word “reset”, but this didn’t work.

**Comment by civier** [comment]
Just to clarify, OzStar is the supercomputer of Swinburne university, where Lizanne is based. However, the problem is general, and has to do with funky behaviour when it comes to keyboard input. e.g., when I use VNM, sometimes pressing a letter on the keyboard causes to the letter to be printed indefinitely.

**Reply by stebo85** [reply]
@civier, can you try a different broweser and report back if you still see the same issues?

**Comment by stebo85** [comment, marked as answer]
Dear @lizannea - thank you so much for alerting us to this issue :) I have never seen this before, but have some ideas where this could come from. 

Could you try a few things to narrow down the problem:
- when you open a new terminal in the virtual desktop environment, is caps lock also on or does it behave normally again?
- if the new terminal works, can you try again connecting to ozStar - does the problem occur again?
- if it occurs again: what command are you using to connect to ozStar?

Cheers
Steffen

**Reply by lizannea** [reply]
Dear @stebo85,

Thank you for helping! 
When I close every thing and start again or try it in a different browser every thing is still in caps. Connecting to OzStar is not the problem per se, as I am able to connect if I type with the shift key down.

**Reply by stebo85** [reply]
Dear @lizannea,

what happens when you press the CAPS Lock key before starting a new session? Does it go back to normal? 

We might have to look at this in a quick interactive zoom session to figure out what's going on :)

Cheers
Steffen

**Reply by lizannea** [reply]
Dear @stebo85, 

That worked! :) 
Thank you.

Cheers, 
Lizanne

**Reply by stebo85** [reply]
ok, great :) Then it's most likely not a bug. You might have just hit the CAPS lock key accidently. Let us know if this behaviour comes up again and in which browser it's happening.

**Comment by stebo85** [comment]
This could be a problem in noVNC. Which browser are you using to open the desktop interface?

**Reply by lizannea** [reply]
I used Chrome

**Reply by stebo85** [reply]
can you try a different browser? There is a bug in chrome that leads to stuck keys with the viewer we are using.
